RecoHGCal/TICL/src/classes_def.xml

```diff
@@ -0,0 +1,10 @@
+
+<lcgdict>
+  <class name="Trackster" ClassVersion="10">
+   <version ClassVersion="10" checksum="1507284825"/>
```

Comment:
please start from version 3. 

In fact, do we even need version tracking for this internal data for the algorithms?
Please consider marking these as transient.
However, if there are plans to keep this saved in any production data files, the class def should be moved to the DataFormats subsystem.